app/controllers/sign_up/completions_controller.rb

```diff
@@ -47,7 +47,7 @@ def completions_presenter
         current_sp: current_sp,
         decrypted_pii: pii,
         requested_attributes: decorated_session.requested_attributes.map(&:to_sym),
-        ial2_requested: sp_session[:ial2] || sp_session[:ialmax],
+        ial2_requested: !!ial2_requested?,
```

Comment:
I would add the boolean cast (`!!`) inside this method rather than outside it?

Reply:
yeahh that is better

Reply:
updated in [43cc375](https://github.com/18F/identity-idp/pull/8350/commits/43cc3759494a001217d59e9372a3c16df540bbde)